ci-operator/step-registry/openshift-qe/ocpbugs-77510-e2e-test/openshift-qe-ocpbugs-77510-e2e-test-commands.sh

```
@@ -0,0 +1,634 @@
+    echo -e "${RED}[$(date '+%Y-%m-%d %H:%M:%S')] ❌ $1${NC}"
+}
+
+# Cleanup function
+cleanup() {
+    # Prevent recursive cleanup calls
+    if [[ "${CLEANUP_RUNNING:-}" == "true" ]]; then
+        return 0
+    fi
+    export CLEANUP_RUNNING=true
+    
+    local exit_code=$?
```

Shellcheck (0.11.0)

[warning] exit_code appears unused. Verify use (or export if used externally).

(SC2034)